static/app/views/starfish/utils/constants.tsx

```diff
@@ -1,4 +1,4 @@
 // This constant is to be used as an arg for `getInterval`.
 // 'metrics' fidelity is intended to match the granularities of stored metrics.
 // This gives us the best/highest fidelity of data for minimum amount of work (don't need to merge buckets).
-export const STARFISH_CHART_INTERVAL_FIDELITY = 'metrics';
+export const STARFISH_CHART_INTERVAL_FIDELITY = 'high';
```

Comment:
will we revert this back for EA? Or is this what we moving forward with?

Reply:
Ended up reverting this, tweaked the `metrics` granularity instead